index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="assets/css/styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <script src="https://use.fontawesome.com/4eb9d85bb2.js"></script>
+</head>
+<body>
+    <header class="portfolio-header">
+        <h1>Jon D-ev</h1>
+        <nav>
+            <ul>
+                <li>
+                    <img src="assets/img/Union.png" alt="menu icon">
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <article class="intro-wrapper">
+        <section class="welcome">
+            <h2 class="greeting-intro">
+                <span class="greeting">Hey there. I'm Jonathan</span>
+                <span class="greeting intro">I'm a software developer</span>
+            </h2>
+            <p class="banner-message">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestitate to contact me</p>
+        </section>
+        <section class="social-media">
+            <ul class="social-media-links">
+                <li><a href="#"><i class="social fa fa-linkedin" aria-hidden="true"></i></a></li>
+                <li><a href=""><i class="social fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a href=""><i class="social fa fa-github" aria-hidden="true"></i></a></li>
+                <li><a href=""><i class="social fa fa-facebook" aria-hidden="true"></i></a></li>
+                <li><a href=""><i class="social fa fa-instagram" aria-hidden="true"></i></a></li>
+            </ul>
+        </section>
```

Comment:
- [ ] Kindly ensure that your design looks exactly like the original design from Figma. For now, your social-media icons don't look like the Figma design. the font-color, margin,... should be the same. Please, solve this issue.

| Figma | Your Design |
|--------|---------------|
|![image](https://user-images.githubusercontent.com/71874425/185642433-077bbbe8-9863-4cac-9603-3db5c8e081a7.png)|![image](https://user-images.githubusercontent.com/71874425/185642758-9174bfcd-0a7e-4424-918b-72ed16a9fff9.png)|
Make sure that you fit the original design.

Reply:
This has been updated. Thanks for this correction 👍